airflow/migrations/versions/e3a246e0dc1_current_schema.py

```diff
@@ -0,0 +1,232 @@
+            'ti_pool',
+            'task_instance',
+            ['pool', 'state', 'priority_weight'],
+            unique=False
+        )
+        op.create_index(
+            'ti_state_lkp',
+            'task_instance',
+            ['dag_id', 'task_id', 'execution_date', 'state'],
+            unique=False
+        )
+        op.create_table(
+            'user',
+            sa.Column('id', sa.Integer(), nullable=False),
+            sa.Column('username', sa.String(length=250), nullable=True),
+            sa.Column('email', sa.String(length=500), nullable=True),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('username')
+        )
+        op.create_table(
+            'variable',
+            sa.Column('id', sa.Integer(), nullable=False),
+            sa.Column('key', sa.String(length=250), nullable=True),
+            sa.Column('val', sa.Text(), nullable=True),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('key')
+        )
+        op.create_table(
+            'chart',
+            sa.Column('id', sa.Integer(), nullable=False),
+            sa.Column('label', sa.String(length=200), nullable=True),
+            sa.Column('conn_id', sa.String(length=250), nullable=False),
+            sa.Column('user_id', sa.Integer(), nullable=True),
+            sa.Column('chart_type', sa.String(length=100), nullable=True),
+            sa.Column('sql_layout', sa.String(length=50), nullable=True),
+            sa.Column('sql', sa.Text(), nullable=True),
+            sa.Column('y_log_scale', sa.Boolean(), nullable=True),
+            sa.Column('show_datatable', sa.Boolean(), nullable=True),
+            sa.Column('show_sql', sa.Boolean(), nullable=True),
+            sa.Column('height', sa.Integer(), nullable=True),
+            sa.Column('default_params', sa.String(length=5000), nullable=True),
+            sa.Column('x_is_date', sa.Boolean(), nullable=True),
+            sa.Column('iteration_no', sa.Integer(), nullable=True),
+            sa.Column('last_modified', sa.DateTime(), nullable=True),
+            sa.ForeignKeyConstraint(['user_id'], ['user.id'], ),
+            sa.PrimaryKeyConstraint('id')
+        )
+        op.create_table(
+            'known_event',
+            sa.Column('id', sa.Integer(), nullable=False),
+            sa.Column('label', sa.String(length=200), nullable=True),
+            sa.Column('start_date', sa.DateTime(), nullable=True),
+            sa.Column('end_date', sa.DateTime(), nullable=True),
+            sa.Column('user_id', sa.Integer(), nullable=True),
+            sa.Column('known_event_type_id', sa.Integer(), nullable=True),
+            sa.Column('description', sa.Text(), nullable=True),
+            sa.ForeignKeyConstraint(['known_event_type_id'],
+                                    ['known_event_type.id'], ),
+            sa.ForeignKeyConstraint(['user_id'], ['user.id'], ),
+            sa.PrimaryKeyConstraint('id')
+        )
```

Comment:
This is awesome -- just want to point out that we added a new table `xcom`. Thanks!